docs/pages/guides/using-sentry.md

```diff
@@ -65,12 +67,25 @@ Sentry.init({
   enableInExpoDevelopment: true,
   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
 });
+```
+
+#### Usage
+
+Depending on which platform you are on (mobile or web), use the following functions in order to capture/catch exceptions:
 
+```js
 // Access any @sentry/react-native exports via:
-Sentry.Native.*
+// Sentry.Native.*
 
 // Access any @sentry/browser exports via:
-Sentry.Browser.*
+// Sentry.Browser.*
+
+// Example:
+try {
+  // your code
+} catch (error) {
+  Sentry.Native.captureException(error);
+}
 ```
```

Comment:
The example then, could be written as:

The following example uses `captureException()` from `Sentry.Native.*` to capture errors:

```js
try {
  // your code
} catch (error) {
  Sentry.Native.captureException(error);
}
```